api/src/pcapi/core/educational/api/stock.py

```diff
@@ -82,7 +83,21 @@ def edit_collective_stock(
     if booking_limit is None:
         check_booking_limit_datetime = stock.bookingLimitDatetime
 
-    offer_validation.check_booking_limit_datetime(stock, check_beginning, check_booking_limit_datetime)
+    booking_status = [booking.status for booking in stock.collectiveBookings]
+    if not stock_data.get("beginningDatetime") and not stock_data.get("bookingLimitDatetime"):
+        price = updatable_fields.get("price")
+        if CollectiveBookingStatus.USED in booking_status:
+            time_left_last_edit = stock.beginningDatetime + datetime.timedelta(days=-2)
+            if stock.bookingLimitDatetime - time_left_last_edit < datetime.timedelta(
+                days=2
+            ) and stock.bookingLimitDatetime - time_left_last_edit >= datetime.timedelta(days=0):
+                if price:
```

Comment:
```suggestion
            last_edition_date = booking.dateUsed + datetime.timedelta(days=2)
            if last_edition_date > datetime.utcnow() and price:
```

on continue a simplifier

Reply:
pas compris d'ou tu sors le booking.dateUsed

Reply:
api/core/educational/models.py:816 ;)